react-validated-input-component/my-app/src/ValidatedInput.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+import { FaCheck } from 'react-icons/fa';
+import { ImCross } from 'react-icons/im';
+
+export function ValidatedInput() {
+  const [password, setPassword] = useState('');
+
+  const length = password.length;
+
+  const includeDigit = /[0-9]/.test(password);
```

Comment:
I think `has` would be a better prefix than `include`. Another good one would be `includes`. The prefix `include` makes it sound like an action rather than like a state.